ipaserver/install/httpinstance.py

```diff
@@ -337,26 +338,26 @@ def __setup_ssl(self):
                 self.start_tracking_certificates()
 
             self.add_cert_to_service()
-
         else:
             if not self.promote:
-                ca_args = [
-                    paths.CERTMONGER_DOGTAG_SUBMIT,
-                    '--ee-url', 'https://%s:8443/ca/ee/ca' % self.fqdn,
-                    '--certfile', paths.RA_AGENT_PEM,
-                    '--keyfile', paths.RA_AGENT_KEY,
-                    '--cafile', paths.IPA_CA_CRT,
-                    '--agent-submit'
-                ]
-                helper = " ".join(ca_args)
-                prev_helper = certmonger.modify_ca_helper('IPA', helper)
+                with tempfile.TemporaryDirectory() as tmpdir:
```

Comment:
ruh roh:
```
2025-02-27T20:00:22Z DEBUG   [11/21]: setting up ssl
2025-02-27T20:00:22Z DEBUG Traceback (most recent call last):
...
  File "/usr/lib/python3.13/site-packages/ipaserver/install/httpinstance.py", line 342, in __setup_ssl
    with tempfile.TemporaryDirectory() as tmpdir:
         ^^^^^^^^
NameError: name 'tempfile' is not defined. Did you mean: 'compile'? Or did you forget to import 'tempfile'?
...
  File "/usr/lib/python3.13/site-packages/ipaserver/install/httpinstance.py", line 342, in __setup_ssl
    with tempfile.TemporaryDirectory() as tmpdir:
         ^^^^^^^^
NameError: name 'tempfile' is not defined. Did you mean: 'compile'? Or did you forget to import 'tempfile'?
```

Reply:
This is fixed. The "remove mod_nss migration" change that was included with this PR was merged upstream so I dropped it from this one. That removed the tempfile import. There was a brief period where this PR was missing the import. You lucked out and found it.